docs/USAGE-GUIDE.md

```diff
@@ -1197,6 +1197,53 @@ int s2n_early_data_cb_async_impl(struct s2n_connection *conn, struct s2n_offered
 
 **s2n_offered_early_data_get_context_length** and **s2n_offered_early_data_get_context** can be called to examine the optional user context associated with the early data. Unlike most s2n-tls callbacks, the context is not configured when the callback is set. Instead, the context is associated with the specific pre-shared key or session ticket used for early data. The context can be set for external pre-shared keys by calling **s2n_psk_set_early_data_context**. For session tickets, **s2n_connection_set_server_early_data_context** can be used to set the context the server includes on its new session tickets. Because the server needs to serialize the context when creating a new session ticket, the context is a byte buffer instead of the usual void pointer.
 
+# Callbacks
+
```

Comment:
I don't think this is worth putting in the guide. We're trying to move away from lists of APIs-- the callbacks are mentioned in the sections on the related features. For example, verify host is mentioned in the ceritifcates section.

Reply:
Sounds good, I was listing all the callbacks for my benefit and thought maybe it would be useful for others also. But it seems like a bit of duplication so I can remove it